embeddings/transformation/pandas_transformation/rename_input_columns_transformation.py

```diff
@@ -0,0 +1,22 @@
+from typing import Dict
+
+import pandas as pd
+
+from embeddings.transformation.transformation import Transformation
+
+
+class RenameInputColumnsTransformation(
```

Comment:
Tests are missing for that transformation

Reply:
Already added